optaplanner-docs/src/main/asciidoc/LocalSearch/LocalSearch-chapter.adoc

```diff
@@ -476,6 +476,46 @@ Use a lower tabu size than in a pure Tabu Search configuration.
   </localSearch>
 ----
 
+[[greatDeluge]]
+== Great Deluge
```

Comment:
Great Deluge must come after Tabu Search, Late Acceptance and Simulated Annealing. I am ok to having it before Step Counting Hill Climbing. It must first earn it's spur before it's promoted to a top position :)

Reply:
Add a row in the table in this section:
https://docs.optaplanner.org/latestFinal/optaplanner-docs/html_single/#optimizationAlgorithmsOverview